lib/engine/game/g_1867/meta.rb

```diff
@@ -17,6 +17,8 @@ module Meta
         GAME_PUBLISHER = :grand_trunk_games
         GAME_RULES_URL = 'https://boardgamegeek.com/filepage/212807/18611867-rulebook'
 
+        KEYWORDS = ['1861'].freeze
```

Comment:
Why inside of 1867?

Reply:
In the code 1861 inherits from 1867, but in the real world, 1867 derived from 1861 so I wanted 1867 to show up when you search for 1861, and this seemed the easiest way to do that.

Maybe it makes more sense to always include a game when you search for any game it derives from, don't know if that would be too complicated to set up yet, but it would let me get rid of this one-off `KEYWORDS`. With the current state of the code, searching for "1846" will show you (among other games) 18 Los Angeles, but searching for "18 Los Angeles" won't show you 1846.